x/evm/client/cli/query.go

```diff
@@ -1,88 +1,105 @@
 package cli
 
 import (
-	"fmt"
-
-	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/client/flags"
-	"github.com/cosmos/cosmos-sdk/codec"
 
+	rpctypes "github.com/cosmos/ethermint/rpc/types"
 	"github.com/cosmos/ethermint/x/evm/types"
 )
 
-// GetQueryCmd defines evm module queries through the cli
-func GetQueryCmd(moduleName string, cdc *codec.Codec) *cobra.Command {
-	evmQueryCmd := &cobra.Command{
+// GetQueryCmd returns the parent command for all x/bank CLi query commands.
+func GetQueryCmd() *cobra.Command {
+	cmd := &cobra.Command{
 		Use:                        types.ModuleName,
 		Short:                      "Querying commands for the evm module",
 		DisableFlagParsing:         true,
 		SuggestionsMinimumDistance: 2,
 		RunE:                       client.ValidateCmd,
 	}
-	evmQueryCmd.AddCommand(flags.GetCommands(
-		GetCmdGetStorageAt(moduleName, cdc),
-		GetCmdGetCode(moduleName, cdc),
-	)...)
-	return evmQueryCmd
+
+	cmd.AddCommand(
+		GetStorageCmd(),
+		GetCodeCmd(),
+	)
+	return cmd
 }
 
-// GetCmdGetStorageAt queries a key in an accounts storage
-func GetCmdGetStorageAt(queryRoute string, cdc *codec.Codec) *cobra.Command {
-	return &cobra.Command{
-		Use:   "storage [account] [key]",
-		Short: "Gets storage for an account at a given key",
+// GetStorageCmd queries a key in an accounts storage
+func GetStorageCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "storage [address] [key]",
+		Short: "Gets storage for an account with a given key and height",
+		Long:  "Gets storage for an account with a given key and height. If the height is not provided, it will use the latest height from context.",
```

Comment:
should we change `account` to `address` in the description as well?
this address will be the hex ethereum address correct? was this previously the cosmos account address passed?